netbox/ipam/api/serializers_/vlans.py

```diff
@@ -110,3 +112,19 @@ class Meta:
     def validate(self, data):
         # Bypass model validation since we don't have a VID yet
         return data
+
+
+class VLANTranslationRuleSerializer(NetBoxModelSerializer):
+
+    class Meta:
+        model = VLANTranslationRule
+        fields = ['policy', 'local_vid', 'remote_vid']
+
+
+class VLANTranslationPolicySerializer(NetBoxModelSerializer):
+    rules = VLANTranslationRuleSerializer(many=True, read_only=True)
+
+    class Meta:
+        model = VLANTranslationPolicy
+        fields = ('id', 'name', 'description', 'rules')
+        brief_fields = ('id', 'rules')
```

Comment:
We should omit rules from the `brief` fields and include the name & description.